package.json

```diff
@@ -28,7 +29,8 @@
     "eslint-plugin-prettier": "^3.1.1",
     "husky": "^3.0.8",
     "lint-staged": "^9.4.2",
-    "prettier": "^1.18.2"
+    "prettier": "^1.18.2",
+    "standard-version": "^6"
```

Comment:
v7 follows npm, which only bumps by minor if it has `BREAKING CHANGE`, which is not what we want